javascript/README.md

```diff
@@ -0,0 +1,129 @@
+# Code Snippets : Javascript
```

Comment:
Keep the heading as "Javascript".  Also, I see that you've added links to the javascript files that we are currently missing in the project folder. May be remove them for now.